File: week-03/day-3/7.py
# create a function that takes a list and returns a new list with all the elements doubled
list = [1, 2, 3, 4]

def elements_double(input):
        newList = []
        newList = input * 2
        return newList

print(elements_double(list))

def elements_double(input):
        newList = []
        for i in input:
            newList = newList + [(i * 2)]
            # newList.append(i*2)
        return newList

print(elements_double(list))

File: week-07/day-02/7.js
'use strict';

var numbers = [2, 5, 11, 29];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function prime(inputArr) {
  return inputArr.every(function (e) {
    if (e < 2) {
      return false;
    }
    for (var i = 2; i < e; i++) {
      if (e % i === 0) {
        return false;
      }
    }
    return true;
  });
}
console.log(prime(numbers));


// var numbers = [2, 5, 11, 29].every(function(e) {
//   var a = Math.random() * 10;
//     return e / a !== 0;
//   //  console.log(e);
// });
//
// console.log(numbers);
